<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addoredit_prepodDiscipline</class>
 <widget class="QDialog" name="addoredit_prepodDiscipline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>317</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Назачение преподавателя на дисциплину</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QListView" name="listView_gruppa"/>
   </item>
   <item row="4" column="1">
    <widget class="QListView" name="listView_sotrudnik"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>будет вести</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_lk">
        <property name="text">
         <string>лекции</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_pr">
        <property name="text">
         <string>практики</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_lb">
        <property name="text">
         <string>лабораторные работы</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Выберите сотрудника</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Выберите группу</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ОК</string>
     </property>
     <property name="icon">
      <iconset resource="../resourses.qrc">
       <normaloff>:/img/img/ok.png</normaloff>:/img/img/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
     <property name="icon">
      <iconset resource="../resourses.qrc">
       <normaloff>:/img/img/cancel.png</normaloff>:/img/img/cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_error">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red</string>
     </property>
     <property name="text">
      <string>Заполните обязательные поля</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
</ui>
